<launch>



<node pkg="sas_robot_driver_franka" type="sas_robot_driver_franka_hand_node" name="franka_hand_1">
<rosparam file="$(find sas_robot_driver_franka)/cfg/sas_robot_driver_franka_hand_1.yaml" command="load"
/>

</node>
</launch>
